app/routes/static.py

```diff
@@ -22,3 +24,10 @@ def privacy():
 @static_blueprint.route("/accessibility", methods=["GET"])
 def accessibility():
     return render_template("static/accessibility")
+
+
+@static_blueprint.route("/json/countries", methods=["GET"])
```

Comment:
This could probably do with a comment to mark it as somewhere that will be deprecated once the relevant file is on the CDN